src/core/state/backendNetworks/backendNetworks.ts

```diff
@@ -18,6 +18,9 @@ import {
 } from '~/core/types/chains';
 import { GasSpeed } from '~/core/types/gas';
 
+import { useRainbowChainsStore } from '../rainbowChains';
+import { useUserChainsStore } from '../userChains';
```

Comment:
I think we may want to use `userChainsStore` directly (and similar for the other hooks) because `/core` components can be consumed outside of the React tree in the other worker script entrypoints. Not sure if Zustand or the compiler would automatically handle that when initializing the hook and calling `getState()`